modules/services.sh

```diff
@@ -326,7 +326,7 @@ function services {
           if [ "$command_option" = "local" ]; then
             for i in $(seq 1 "$(get_port $service_name | wc -l)")
             do
-              local_url=$(hostname -I | head -n1 | cut -d " " -f1)
+              local_url=$(treehouses networkmode info | grep -oP -m1 '(?<=ip: ).*?(?=,)')
```

Comment:
Can't you just do this with ```networkmode "info"``` ? Cli.sh has these files included. On a manual or separate installation.  I think might not have access to ```treehouses``` in command line. 
https://github.com/treehouses/cli/blob/master/cli.sh#L208-L209

Reply:
Yeah, I always forget NOT to use `treehouses`.